packages/contracts/src/contracts/current/protocol/Exchange/libs/LibOrder.sol

```diff
@@ -21,9 +21,16 @@ pragma solidity ^0.4.24;
 contract LibOrder {
 
     bytes32 constant DOMAIN_SEPARATOR_SCHEMA_HASH = keccak256(
-        "DomainSeparator(address contract)"
+        "DomainSeparator(",
+        "string name,",
+        "string version,",
+        "address contract",
+        ")"
     );
 
+    bytes32 constant DOMAIN_SEPARATOR_NAME_HASH = keccak256("0x Protocol");
+    bytes32 constant DOMAIN_SEPARATOR_VERSION_HASH = keccak256("1");
```

Comment:
Shouldn't this be `2`?

Reply:
This number represents our EIP712 schema, not the protocol version.

Reply:
If you could add a comment to that effect that would be great.